### PR TITLE
Do not add 'domainid' to documents. Now will post everything to the 'do…

### PR DESCRIPTION
**Pull request by angelRev**
…mainid' set in params. 'forcedomainid' no longer used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +37,7 @@ mindmeld-poster -t <admin token>  <dir>
 
 To read from stdin:
 ```sh
-cat myfile | mindmeld-poster -t <admin token>
```
Comment (jjackson26) [inline README.md]: Does this work? I thought we need to specify the domain id here.

**Comment by jjackson26** [comment]
Looks good but let's fix the tests

File: README.md (hunk 1 above)
Comment (angelRev) [inline README.md]: I just changed this in the whole README to reflect the new use.

**Comment by jjackson26** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (swarajban) [inline README.md]: should work, the poster can read from STDIN

Comment (swarajban) [inline README.md]: ...but haven't actually tried myself :smile:

**Pull request merged**